packages/home-service/src/schema.ts

```diff
@@ -1,6 +1,13 @@
 import { Document, Model, model, Schema } from 'mongoose';
 
 export const HomeSchema: Schema = new Schema({
```

Comment:
@sayak-sarkar @riginoommen Should this be called MicroserviceSchema?

HomeSchema is not helpful and is incorrect.

Reply:
I dont think so `MicroserviceSchema` is not helping

Reply:
Some other name then. Something other than HomeSchema.

I said MicroserviceSchema because look at the data that's being stored in it. It is the details of the in-build microservices. Not the details of the Home/Homepage.

Reply:
@deshmukhmayur It's not just details of built-in microservices it's also SPAs
```
{
    name: 'Name/Title'
    description: 'Description of the entity,
    link: 'Link to entity'
    icon: 'Icon for the entity',
    entityType: 'Spa/Microservice',
    colorScheme: 'In case of microservice entity',
    videoUrl: 'In caase of spa entity',
}
```
It's information to be displayed on home-page

Reply:
Yeah okay. But my question still stands, how does HomeSchema describe that?

Reply:
Just because it's going to be displayed on the Home page does not mean it should be named Home.
Maybe call it `HomeServiceSchema` if you want to keep Home in there. My only suggestion is that it needs to describe the kind of data that will be stored in it.

Reply:
Changed the name `HomeServiceSchema`